src/ACEtk/block/SecondaryParticleEnergyDistributionBlock/src/generateFrames.hpp

```diff
@@ -0,0 +1,13 @@
+void generateFrames() {
+
+  if ( this->TYRH() ) {
+
+    this->frames_.insert( this->frames_.end(),
+                          this->TYRH()->referenceFrames().begin(),
+                          this->TYRH()->referenceFrames().begin() + this->NR() );
```

Comment:
I believe the code to be correct, but clang-format is sure that this is an un-checked access of the optional. Maybe it thinks that `TYRH()` can change state?

Anyways just saying something in case you see something.